core/modules/memman/MemFile.h

```diff
@@ -50,7 +50,7 @@ class MemFile {
     struct MLResult {
         uint64_t bLocked;
         int      retc;
-        MLResult() {}
+        MLResult() : bLocked{false}, retc(0) {}
```

Comment:
Oh yes, Travis didn’t like your code at all. Why is that?

From: Fritz Mueller 
Sent: Thursday, August 11, 2016 2:27 PM
To: lsst/qserv 
Cc: Andrew Hanushevsky ; Comment 
Subject: Re: [lsst/qserv] Minor fixes for issues flagged by Eclipse codan (#246)

In core/modules/memman/MemFile.h:

> @@ -50,7 +50,7 @@ class MemFile {
>      struct MLResult {
>          uint64_t bLocked;
>          int      retc;
> -        MLResult() {}
> +        MLResult() : bLocked{false}, retc(0) {}
Yup, thanks for catching this -- I got thrown by the hungarian!

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or mute the thread.